Dockerfile

```diff
@@ -1,47 +1,42 @@
-FROM python:3.7-buster
+FROM node:8 as node_build
+
+COPY package.json package-lock.json ./
+COPY npm_install_wrapper.sh npm_install_wrapper.sh ./
+RUN ./npm_install_wrapper.sh
+
+COPY . .
```

Comment:
This should be fine, just wondering if there is a way to be more explicit in its path, Sonar may be okay with it.

Reply:
Sonar isn't ok with it - I figured it was ok as we then do the copy later on of just the assets but would like to make this more explicit if we can.

Reply:
After have a discussion around this in person on a call, this should be fine.